### DISCUSSION TITLE
Oracle Java based application license clarification

### DISCUSSION THREAD

**Discussion by itsjitendra** [freeplane/freeplane#1921] [Help]
I have installed Freeplane application on my computer and planning to rollout to multiple computers. I can see its Java based since Java is license based however this application is free GPL license, Could you please clarify if Oracle will not claim any charges or there will be no additional package needs to be installed for using Freeplane application.

**Comment by dpolivaev** [comment]
You can use any open Java distribution.
Freeplane distributions including Java are based on Azul builds, see https://www.azul.com/products/core/openjdk-terms-of-use/
